blog/2017-01-07-monthly-release-cadence.md

```diff
@@ -0,0 +1,25 @@
+---
+title: A new Release Cadence: Releasing December and January RC
```

Comment:
The title refers to a new release cadence, but doesn't specify the frequency :)

Maybe something like "Adopting a monthly Release Cadence: December and January RC now available" though that feels a bit long.